index.html

```diff
@@ -1,16 +1,90 @@
 <!DOCTYPE html>
 <html lang="en">
-
-<head>
-    <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
     <title>Responsive Cake webpage</title>
     <!-- Add a link to your css file here -->
-</head>
+    <link rel="stylesheet" href="style.css" />
+    <script src="scripts.js"></script>
+  </head>
 
-<body>
+  <body>
     <!-- Add your markup here -->
-</body>
+    <header class="header" style="height: 32vh">
+      <div class="cake_shop_avatar">
+        <!-- <img src="img/cupcake.webp" /> -->
+      </div>
+      <div class="cake_shop_slogan">
+        <h1>The best cakes in town delivered to your door</h1>
+      </div>
+
+      <nav class="navbar">
+        <ul class="nav_menu">
+          <li class="nav_item"><a class="nav_link" href="#">Home</a></li>
+          <li class="nav_item"><a class="nav_link" href="#">Cakes</a></li>
+          <li class="nav_item"><a class="nav_link" href="#"> Ordering</a></li>
+          <li class="nav_item"><a class="nav_link" href="#">Lessons</a></li>
+          <li class="nav_item"><a class="nav_link" href="#">About</a></li>
+        </ul>
+      </nav>
+    </header>
+
+    <section class="main">
+      <article class="hero">
+        <div class="welcome_text">
+          <h1>Welcome</h1>
+          <p>
+            Lorem ipsum, dolor sit amet consectetur adipisicing elit. Porro,
+            quam hic! Neque recusandae consequatur fugit alias quibusdam!
+            Perspiciatis enim ab facere possimus similique, aut at pariatur
+            soluta quis reiciendis rem.
+          </p>
+        </div>
+        <div class="welcome_image">
+          <img src="img/Cake-.jpeg" alt="cake" width="300" height="300" />
```

Comment:
Ditto, it is better to use css to configure style, and you probably want to avoid hard coding width and height for it to be responsive

Reply:
Hard coding normally means writing special numbers into your code that is hard/impossible to change without changing code

Reply:
Thank you for the information.